script/foreman-debug

```diff
@@ -79,6 +79,15 @@ add_cmd() {
   eval $CMD >> "$DIR/$OUT" 2>&1
 }
 
+# add outout of the sql and redirect possible errors there
+add_sql() {
+  SQL=$1
+  OUT=$2
+  printv " - $OUT"
+  echo -e "SQL> $SQL\n" > "$DIR/db_$OUT"
+  echo "$SQL" | su postgres -c 'psql foreman' >> "$DIR/db_$OUT" 2>&1
```

Comment:
Yes, this is PostgreSQL only, I will add a check. Username is not relevant tho, it's actually database name here, we are accessing this from the superuser.